apps/examples/smp/smp_main.c

```diff
@@ -296,7 +301,24 @@ int smp_main_prthread(int argc, FAR char *argv[])
 	}
 
 
+#ifdef CONFIG_AMP
+	/* If AMP is enabled, then we will run the first CONFIG_TESTING_SMP_NBARRIER_THREADS / 2
+	 * threads on core 1 and remaining on default core
```

Comment:
This example is meant for all chipsets, you should not hardcode it like this way. Please utilize CONFIG_SMP_NCPUS, and divide the barrier threads evenly. For example, imx6 supports 4 cortex-A7 cores. Then, each core could handle 25% of workload.

Reply:
Done.